christfellowshipchurch/web-app-v2
# Replaced Westlake Images

### About
This PR replaces the old construction Westlake campus location images with new ones (see ticket below for more details) 

### Test Instructions
Open the preview link + /locations/westlake and you should see the new carousel images, image 2 and 5 were updated

### Closes Tickets
[CFDP-2464](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2464)

### Screenshots
![Screen Shot 2023-02-23 at 1 41 18 PM](https://user-images.githubusercontent.com/46769629/221000616-82353b1d-0e9c-4ec7-baf3-043c209a9c8f.png)

![Screen Shot 2023-02-23 at 1 39 18 PM](https://user-images.githubusercontent.com/46769629/221000151-fc7b8df1-2c65-44c8-a2a8-9deb4a33aa1a.png)

[CFDP-2464]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2464?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
